docs/02-View Logs/01-Navigating-to-other-pages..md

```diff
@@ -15,3 +17,19 @@ The Parsley log viewer consists of a few major components listed below:
 **Details menu:** The menu provides a variety of togglable settings which can affect the log view as well as the functionality of searching and filtering. These settings will be discussed in detail in further sections. The menu also provides external links to the raw and HTML logs.
 
 Each component is also covered in detail in further sections. 
+
+The task page link above the log links to the task page on Spruce. 
+
+![task page](https://user-images.githubusercontent.com/624531/207638552-0119b374-2e16-490b-80a2-e1cdab1dd21d.png)
+
+Raw and HTML buttons in the details menu links to the raw and HTML versions of the current log. 
+
+![raw and html](https://user-images.githubusercontent.com/624531/207638591-69a596d4-9578-40b3-8dbf-3fc84230040a.png)
+
+Job Logs Page
+
+Resmoke.py – the test runner for mongodb-mongo-* tests – has a concept of a 'job' for organizing groups of tests to run. mongodb-mongo* logs include a “Job Logs” button that allows you to view all logs within a single job.  
```

Comment:
I copied and pasted this content from the deleted page without editing the content at all and didn't plan to make copy edits (since that would be a big job on its own).. however if there's a specific change you want to see here can you please suggest it so I can see what you mean? Not sure what you meant about em dashes

Reply:
Ah, okay! I missed this was copy pasted. I'll fix separately. I meant that – should be —.